libs/docs/core/popover/examples/popover-closing-example/popover-closing-example.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="fd-docs-flex-display-helper">
+    <fd-popover [closeOnOutsideClick]="false" [focusAutoCapture]="true" [focusTrapped]="true">
+        <fd-popover-control>
+            <button fd-button>preventCloseOnOutsideClick</button>
+        </fd-popover-control>
+        <fd-popover-body [style.min-width.px]="400">
+            <div fd-popover-body-header id="popover-bar-simple-header-1">
+                <div fd-bar barDesign="header">
+                    <div fd-bar-left>
+                        <fd-button-bar
+                            glyph="navigation-left-arrow"
+                            fdType="transparent"
+                            ariaLabel="Back"
+                        ></fd-button-bar>
+                        <fd-bar-element><h5 fd-title>Header</h5></fd-bar-element>
+                    </div>
+                </div>
+            </div>
+            <fd-avatar [circle]="true" [style.margin.rem]="1" size="xl" glyph="group"></fd-avatar>
+        </fd-popover-body>
+    </fd-popover>
+    <fd-popover [closeOnEscapeKey]="false" [focusAutoCapture]="true" [focusTrapped]="true">
+        <fd-popover-control>
+            <button fd-button>preventCloseOnEscapeKey</button>
+        </fd-popover-control>
+        <fd-popover-body [style.min-width.px]="400">
+            <div fd-popover-body-header id="popover-bar-simple-header-1">
+                <div fd-bar barDesign="header">
+                    <div fd-bar-left>
+                        <fd-button-bar
+                            glyph="navigation-left-arrow"
+                            fdType="transparent"
+                            ariaLabel="Back"
+                        ></fd-button-bar>
+                        <fd-bar-element><h5 fd-title>Header</h5></fd-bar-element>
+                    </div>
+                </div>
+            </div>
+            <fd-avatar [circle]="true" [style.margin.rem]="1" size="xl" glyph="group"></fd-avatar>
+        </fd-popover-body>
+    </fd-popover>
+    <fd-popover
+        [closeOnEscapeKey]="false"
+        [closeOnOutsideClick]="false"
+        [focusAutoCapture]="true"
+        [focusTrapped]="true"
+    >
+        <fd-popover-control>
+            <button fd-button>preventCloseOnEscapeKey & preventCloseOnEscapeKey</button>
```

Comment:
I think this is supposed to say `preventCloseOnEscapeKey & preventCloseOnOutsideClick`

Reply:
But also maybe just go with typically formatted text i.e. "Prevent Close on Escape Key and Prevent Close on Outside Click" because the camel case on these buttons makes me think it's a property or something that the developer can set